deviations/static/deviations/remove_deviation.js

```diff
@@ -0,0 +1,20 @@
+$('table form').on('submit', function(event) {
```

Comment:
The script should be wrapped inside `$(function() { .... });` so that it waits for the DOM to finish loading.

The selector `table form` could be made more specific. E.g., `table[data-deviations-list]` if we add a data attribute to the table in the HTML template.